cmd/zpool/zpool_main.c

```diff
@@ -3549,6 +3549,7 @@ zpool_do_import(int argc, char **argv)
 	boolean_t do_scan = B_FALSE;
 	boolean_t pool_exists = B_FALSE;
 	boolean_t pool_specified = B_FALSE;
+	boolean_t missing_log = B_FALSE;
```

Comment:
Would ignore_missing_log be more descriptive?

Reply:
Thanks but I think the name `missing_log` is aligned with the flag `ZFS_IMPORT_MISSING_LOG`.

Reply:
No problem sounds good